### PR TITLE
Create Simulation.md

### PR DESCRIPTION
**Pull request by syuanuvt**
Written by Shuai from MTO :)

### CODE REVIEW

File: Simulation.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Overview of the steps
+Write the code -> Test the code -> Set the desired parameters -> Run the simulation
+# Write the code
+Write the code with your brain and heart. Do use Github to control the versions, since you are using it now.
+# Test the code
+Code testing is always painful, but the Aha moments alwasy pay off. Just like your life.
```
Comment (keloalex) [inline Simulation.md:6]: Hahah, "Ah hah moments pay off"

File: Simulation.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+# Set the desrired parameters
+Two ways to confirm the parameter settings you should use. (1) Following what others do in the previous literature 
+and (2) Following what your supervisors suggest. You should know that only (2) could help you to get rid of your supervisors' angry faces.
+# Run the simulation
+All about running simulation is wait, wait, and wait. Why not go for a walk on the mountain tops?
```
Comment (keloalex) [inline Simulation.md:11]: There are no mountains in Tilburg

File: Simulation.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Overview of the steps
+Write the code -> Test the code -> Set the desired parameters -> Run the simulation
+# Write the code
+Write the code with your brain and heart. Do use Github to control the versions, since you are using it now.
+# Test the code
+Code testing is always painful, but the Aha moments alwasy pay off. Just like your life.
+# Set the desrired parameters
+Two ways to confirm the parameter settings you should use. (1) Following what others do in the previous literature 
+and (2) Following what your supervisors suggest. You should know that only (2) could help you to get rid of your supervisors' angry faces.
```
Comment (keloalex) [inline Simulation.md:9]: Supervisors never have angry faces, unless they compare Paris and Tilburg...

**Comment by svanhalem** [comment]
merge this one?

**Pull request merged**